cmd/startupapicheck/pkg/check/api/api.go

```diff
@@ -0,0 +1,146 @@
+/*
+Copyright 2021 The cert-manager Authors.
```

Comment:
I wonder if the year needs updating on these files?
Perhaps something to put on the list for 2024 given that is relatively close :)

Reply:
I think our general policy on this has been to not bother because I don't think there's been any legal reason to do so. It's a fair comment that these new files might as well say $CURRENTYEAR though - I'll change that because why not!